packages/ui/molecules/Readmore/Readmore.twig

```diff
@@ -0,0 +1,65 @@
+{#
+/**
+ * @file
+ *   Readmore component.
+ *
+ * @param array $attr
+ *   Use it to customize the root element attributes.
+ * @param string main_content $main_content
+ *   Define the main content.
+ * @param array $main_content_attr
+ *   Custom attributes for the main content.
+ * @param string $hidden_content
+ *   Define the hidden content.
+ * @param array $hidden_content_attr
+ *   Custom attributes for the hidden content.
+ * @param array {label: string, label2: string} $btn
+*   Define the button.
+ * @param array $btn_attr
+ *   Custom attributes for the button.
+ * @param boolean $animation
+ *   Wether to use an animation or not
+ * @param number $length
+ *   The maximum characters length before hidding the content if only one content is provided.s
+ */
+#}
+
+{% set attributes =
+  merge_html_attributes(attr ?? null, { data_component: 'Readmore', data_option_animate: ' true' })
+%}
+
+{% set main_content_attributes =
+  merge_html_attributes(
+    main_content_attr ?? null,
+    { class: 'optionnel' },
+    { data_ref: 'main_content', class: ['main_content'] }
+  )
+%}
+
+{% set hidden_content_attributes =
+  merge_html_attributes(
+    hidden_content_attr ?? null,
+    { class: 'optionnel' },
+    { data_ref: 'hidden_content', aria_hidden: 'true', class: ['hidden_content hidden'] }
+  )
+%}
+
+{% set btn_attr =
+  merge_html_attributes(btn_attr ?? null, { class: 'optionnel' }, { class: ['button'] })
+%}
+
+{% set animation = animation|default(true) %}
+{% set length = length|default('70') %}
+
+<div {{ html_attributes(attributes) }}>
+  <div {{ html_attributes(main_content_attributes) }}>
+    {{ main_content }}
+  </div>
+  <div {{ html_attributes(hidden_content_attributes) }}>
+    {{ hidden_content }}
```

Comment:
**suggestion:** Wrap the content in a block 

We can use more advanced markup when working with a block vs. a parameter.